system-test/testnet-automation.sh

```diff
@@ -179,6 +179,18 @@ function launch_testnet() {
   SLOTS_PER_SECOND="$(bc <<< "scale=3; ($END_SLOT - $START_SLOT)/($SLOT_COUNT_END_SECONDS - $SLOT_COUNT_START_SECONDS)")"
   execution_step "Average slot rate: $SLOTS_PER_SECOND slots/second over $((SLOT_COUNT_END_SECONDS - SLOT_COUNT_START_SECONDS)) seconds"
 
+  sleep 10
+  ALIVE_SLOT=$(get_slot)
+  execution_step "Checking if root slot is still advancing - Slot: $END_SLOT, Slot after 10 seconds: $ALIVE_SLOT"
+
+  if [[ "$(bc <<< "scale=0; ($ALIVE_SLOT - $END_SLOT)")" -gt 0 ]]; then
```

Comment:
Avoiding `bc` seems possible here though:
```suggestion
  if [[ "$ALIVE_SLOT" -gt "$END_SLOT" ]]; then
```